lib/main.ts

```diff
@@ -45,6 +45,7 @@ export interface TranspilerOptions {
 
 export const COMPILER_OPTIONS: ts.CompilerOptions = {
   allowNonTsExtensions: true,
+  experimentalDecorators: true,
```

Comment:
Generally LGTM, but shouldn't this be a change in Angular that gets sent here via the options passed in the ctor?